### PR TITLE
feat(medusa): Allow empty fields/expand

### PR DESCRIPTION
**Pull request by adrien2p**
**What**

Allow to pass an empty fields/expand as the query params. This allows a user to only select an entity property and no relations if wanted. Or only relations without fields, unless there is an order and in that case typeorm select the column to be able to order and therefore returns it.

Screenshot 2023-02-09 at 14 40 49

FIXES CORE-1111
FIXES CORE-1113

### CODE REVIEW

**Comment by adrien2p** [comment]
@shahednasser just to let you know that now the expand query params allows an empty value if the user does not want any relations to be returned

**Comment by olivermrbl** [comment]
@adrien2p We seem to have an issue with the integration tests 🤔

**Comment by adrien2p** [comment]
Yes i am on it, trying ti identify the culprit

**Comment by adrien2p** [comment]
It was green before the last commit an now it is not

**Comment by olivermrbl** [comment]
@srindom Looping you in to also sign off on this one :)

**Review by olivermrbl — APPROVED** [review]
LGTM - pending @srindom approval

**Review by fPolic — APPROVED** [review]
LGTM

**Review by srindom — APPROVED** [review]
LGTM!

**Pull request merged**
